File: tasks/webServer/cgi-bin/full_name.py
#!/usr/bin/python3

import cgi,cgitb
cgitb.enable()

form = cgi.FieldStorage(encoding="utf-8")

first_name = form.getvalue('first_name') or ''
last_name  = form.getvalue('last_name') or ''


print("Content-type:text/html\r\n\r\n")
print("<html>")
print("<head>")
print("<title>Second CGI Program</title>")
print("</head>")
print("<body>")
print("<h2>Full Name: {} {}</h2>".format(first_name,last_name))
print("</body>")
print("</html>")

File: tasks/webServer-perl/cgi-bin/full_name.pl
#!/usr/bin/perl
use CGI qw(:standard);
my $query = new CGI;

my $first_name = $query->param('first_name');
my $last_name = $query->param('last_name');



print "<h2>Hello, $first_name $last_name</h2>";


1;
